html/workout.html

```diff
@@ -7,28 +7,170 @@
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lato">
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Poppins">
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Raleway">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+
     <title>Workout - Motivation Co.</title>
 </head>
 
 <body>
     <!--NAVIGATION BAR BEGINS-->
     <header>
+
         <div class="nav-bar">
+
             <div class="left-nav-bar">
-                <a href="../index.html" style="text-decoration: none;"><h2>Motivation Co.</h2></a>
+                <a href="index.html" style="text-decoration: none;"><h2>Motivation Co.</h2></a>
                 <img src="../imgs/gearicon.png" width="25" height="25" alt="gear icon">
             </div>
             <nav>
                 <ul>
-                    <li><a href="../index.html">Home</a></li>
+                    <li><a href="index.html">Home</a></li>
                     <li><a href="blog.html">Blog</a></li>
                     <li><a href="workout.html" class="current-page">Workout</a></li>
                     <li><a href="contact.html">Contact</a></li>
                 </ul>
             </nav>
         </div>
+
     </header>
     <!--NAVIGATION BAR ENDS-->
+
+    
+    <div class="main-container">
+        
+        <main>
+
+            <div id="form1">

```

Comment:
For the huge scope of this div, it would be better if "form1" is a class. Follow this change to a class in the CSS file too.